pom.xml

```diff
@@ -181,6 +181,8 @@
         <gitHubRepo>jenkinsci/trilead-ssh2</gitHubRepo>
         <!-- TODO fix medium-threshold violations -->
         <spotbugs.threshold>High</spotbugs.threshold>
+        <source>17</source>
```

Comment:
This is inherited from Jenkins plugin; it should not be defined here.

Reply:
Restored pom to original.